### DISCUSSION TITLE
How can i include urls to my django app ?

### DISCUSSION THREAD

**Discussion by Ekata2003** [Ekata2003/Social-media-Django#11] [Q&A] [answered]
How can i include urls to my django app ?

**Comment by prasad-chavan1** [comment, marked as answer]
# Including URLs in Your Django App

To include URLs in your Django app, follow these steps:

1. **Create a Django App**:
   - Create a Django app if you haven't already using `python manage.py startapp myapp`.

2. **Define URLs**:
   - In your app's `urls.py` file, define URL patterns for views.
   
   ```python
   from django.urls import path
   from . import views

   urlpatterns = [
       path('myapp/', views.my_view, name='my_view'),
   ]
